engine/gamesys/src/gamesys/components/comp_mesh.cpp

```diff
@@ -235,6 +235,8 @@ namespace dmGameSystem
         world->m_WorldVertexData = 0x0;
         world->m_WorldVertexDataSize = 0;
 
+        world->m_GraphicsContext = dmRender::GetGraphicsContext(context->m_RenderContext);
```

Comment:
Before changes with GL handles `NewVertexBuffer` doesn't care about graphics_context pointer. That's why it worked before.